lisa_commands/request_helper.py

```diff
@@ -192,7 +192,7 @@ def gify_search(media_type, query):
     :query what to query for
     """
 
-    limit = 10
+    limit = 3
```

Comment:
Is there a reason to have gif limit different than other image searching? I imagine the use cases are almost identical.

Reply:
Seconding this, gif searches can probably be a bit more lax.